Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-2064.dfy.expect

```diff
@@ -0,0 +1,35 @@
+git-issue-2064.dfy(35,9): Error: to be a refinement of abstract type 'UnitTestParent.T', type synonym 'OpaqueTypeMixedUp.T' must support the type characteristic "no references"
+git-issue-2064.dfy(36,9): Error: to be a refinement of abstract type 'UnitTestParent.U', type synonym 'OpaqueTypeMixedUp.U' must support the type characteristic "equality"
+git-issue-2064.dfy(37,9): Error: to be a refinement of abstract type 'UnitTestParent.V', type synonym 'OpaqueTypeMixedUp.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(51,9): Error: to be a refinement of abstract type 'UnitTestParent.T', type synonym 'Arrow.T' must support the type characteristic "no references"
+git-issue-2064.dfy(52,9): Error: to be a refinement of abstract type 'UnitTestParent.U', type synonym 'Arrow.U' must support the type characteristic "equality"
+git-issue-2064.dfy(53,9): Error: to be a refinement of abstract type 'UnitTestParent.V', type synonym 'Arrow.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(54,9): Error: to be a refinement of abstract type 'UnitTestParent.W', type synonym 'Arrow.W' must support the type characteristic "nonempty"
+git-issue-2064.dfy(60,9): Error: to be a refinement of abstract type 'UnitTestParent.U', type synonym 'ArrowGood.U' must support the type characteristic "equality"
+git-issue-2064.dfy(66,9): Error: to be a refinement of abstract type 'UnitTestParent.T', type synonym 'ArrowDirect.T' must support the type characteristic "no references"
+git-issue-2064.dfy(67,9): Error: to be a refinement of abstract type 'UnitTestParent.U', type synonym 'ArrowDirect.U' must support the type characteristic "equality"
+git-issue-2064.dfy(68,9): Error: to be a refinement of abstract type 'UnitTestParent.V', type synonym 'ArrowDirect.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(69,9): Error: to be a refinement of abstract type 'UnitTestParent.W', type synonym 'ArrowDirect.W' must support the type characteristic "nonempty"
+git-issue-2064.dfy(74,9): Error: to be a refinement of abstract type 'UnitTestParent.T', type synonym 'Class.T' must support the type characteristic "no references"
+git-issue-2064.dfy(76,9): Error: to be a refinement of abstract type 'UnitTestParent.V', type synonym 'Class.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(77,9): Error: to be a refinement of abstract type 'UnitTestParent.W', type synonym 'Class.W' must support the type characteristic "nonempty"
+git-issue-2064.dfy(81,10): Error: to be a refinement of abstract type 'UnitTestParent.T', class 'ClassDirect.T' must support the type characteristic "no references"
+git-issue-2064.dfy(83,10): Error: to be a refinement of abstract type 'UnitTestParent.V', class 'ClassDirect.V' must support the type characteristic "auto-initialization"
```

Comment:
```suggestion
git-issue-2064.dfy(83,10): Error: to be a refinement of abstract type 'UnitTestParent.V' declared `(0)`, class 'ClassDirect.V' would need to support auto-initialization, which non-null objects do not support - you might want to refine `type V = ClassName?" for your class, so that `null` can be used to auto-initialize the type `V`
```
Otherwise, the question our user would have is "then how do I make my class auto-initialize" which is a dead end :-)

Reply:
I did not act on this suggest or the next two. These three suggestions would apply if the refinement module use a type synonym and the RHS of that type synonym is a type for which we can offer a hint. While I do like error messages that suggest what to do, I'm unconvinced these hints are worth the added logic in the Dafny implementation. But if you insist on these hints, push back and I'll reconsider.